0.4/examples/image-label_strict/colors_properties.json

```diff
@@ -0,0 +1,29 @@
+{
+  "image-label": {
+    "version": "0.4",
+    "colors": [
+      {
+        "label-value": 1,
+        "rgba": [255, 255, 255, 1]
```

Comment:
The `alpha: 1` here makes it look like the `alpha` is a fraction (as in css) which isn't what the spec defines. `255` would make a better (less confusing) example.

Reply:
But the second item in the array with alpha value `128` should make this unambiguous no?
Easy enough to invert and/or change values